script.js

```diff
@@ -0,0 +1,602 @@
+            }else if(russianKeys[key] == "Tab"){
+                button.classList.add("big");
+                button.classList.add("tab");
+            }else if(typeof(russianKeys[key]) == "number"){
+                button.classList.add("number");
+            }else if(russianKeys[key] == "Space"){
+                button.classList.add("space");
+                button.addEventListener("click", () => {
+                    textarea.value += " ";
+                    textarea.focus();
+                })
+            }else if(russianKeys[key] == "Control"){
+                button.classList.add("control");
+            }else if(russianKeys[key] == "Delete"){
+                button.classList.add("delete");
+                button.addEventListener("click", () => {
+                    textarea.setRangeText('', textarea.selectionStart, textarea.selectionEnd + 1);
+                    textarea.focus();
+                  });
+            }else if(russianKeys[key] == "Alt"){
+                button.classList.add("alt");
+            }else{
+                button.classList.add("key");
+                button.addEventListener("click", () => {
+                    textarea.value += button.innerHTML; 
+                    textarea.focus();
+                })
+                
+            }
+            button.innerHTML = `${russianKeys[key]}`
+            fragment.append(button);
+        }
+        
+    }
+    keyBoard.append(fragment);
+}
+
+drawButtons(lang)
+
+const upperCase = (l) => {
+    keyBoard.innerHTML = "";
+    const fragment = document.createDocumentFragment();
+    if(l == "eng"){
+        for(let key in englishKeysUp){
+            const button = document.createElement("button");
+            if(englishKeysUp[key] == "Backspace" || englishKeysUp[key] == "Enter" || englishKeysUp[key] == "Tab"){
+                button.classList.add("big");
+            }else if(englishKeysUp[key] == "Shift"){
+                button.classList.add("big");
+                button.classList.add("active");
+            }else if(englishKeys[key] == "CapsLock"){
+                button.classList.add("big");
+                button.classList.add("capslock");
+            }else if(typeof(englishKeysUp[key]) == "number"){
+                button.classList.add("number");
+            }else if(englishKeysUp[key] == "Space"){
+                button.classList.add("space");
+            }else{
+                button.classList.add("key");
+                button.addEventListener("click", () => {
+                    textarea.value += button.innerHTML;
+                    if(shift){
+                        shift = false
+                        drawButtons(lang) 
+                    }  
+                })
+            }
+            button.innerHTML = `${englishKeysUp[key]}`
+            fragment.append(button);
+        }
+    }else if(l == "rus"){
+        for(let key in englishKeysUp){
+            const button = document.createElement("button");

```

Comment:
Please also create a function that will do that work as mentioned above.
Do not repeat yourself.